packages/help/src/__snapshots__/Help.spec.js.snap

```diff
@@ -2,15 +2,35 @@
 
 exports[`<Help> Render Should render Help component 1`] = `
 <span
+  onClick={[Function]}
   onMouseEnter={[Function]}
   onMouseLeave={[Function]}
 >
   <div
     className="af-popover__container"
   >
     <div
-      className="af-popover__container-over"
+      classModifier={null}
+      className={null}
```

Comment:
it's seem to miss the prop className in your snapshot

Reply:
I was destructuring props where I shouldn't have so it override the className